src/frontend/cellprofiler/modules/measureobjectsizeshape.py

```diff
@@ -826,51 +370,13 @@ def display(self, workspace, figure):
             title="default",
         )
 
-    def perform_measurement(self, workspace, function, object_name, feature_name):
-        """Perform a measurement on a label matrix
-
-        workspace   - the workspace for the run
-        function    - a function with the following sort of signature:
-                      image - an image to be fed into the function which for
-                              our case is all ones
-                      labels - the label matrix from the objects
-                      index  - a sequence of label indexes to pay attention to
-        object_name - name of object to retrieve from workspace and deposit
-                      in measurements
-        feature_name- name of feature to deposit in measurements
-        """
-        objects = workspace.get_objects(object_name)
-        if len(objects.indices) > 0:
-            data = objects.fn_of_label_and_index(function)
-        else:
-            data = numpy.zeros((0,))
-        self.record_measurement(workspace, object_name, feature_name, data)
-
-    def perform_ndmeasurement(self, workspace, function, object_name, feature_name):
-        """Perform a scipy.ndimage-style measurement on a label matrix
-
-        workspace   - the workspace for the run
-        function    - a function with the following sort of signature:
-                      image - an image to be fed into the function which for
-                              our case is all ones
-                      labels - the label matrix from the objects
-                      index  - a sequence of label indexes to pay attention to
-        object_name - name of object to retrieve from workspace and deposit
-                      in measurements
-        feature_name- name of feature to deposit in measurements
-        """
-        objects = workspace.get_objects(object_name)
-        if len(objects.indices) > 0:
-            data = objects.fn_of_ones_label_and_index(function)
-        else:
-            data = numpy.zeros((0,))
-        self.record_measurement(workspace, object_name, feature_name, data)
-
     def record_measurement(self, workspace, object_name, feature_name, result):
         """Record the result of a measurement in the workspace's measurements"""
         data = centrosome.cpmorphology.fixup_scipy_ndimage_result(result)
```

Comment:
Should this be in library? It's a "mere" fixup, but it's also a morphological operation, and users of library may find it convinient to have via library.